Automattic/wp-calypso
# Add itemized discounts to Jetpack price breakdown

## Proposed Changes

We want to make it more obvious to see how much user saves by introduction and multi-year offers:

![CleanShot 2024-01-12 at 15 56 21@2x](https://github.com/Automattic/wp-calypso/assets/8419292/e2e40eb2-22a0-4dbc-9acb-c147a6516f7e)

Formula for calculating introduction offer for 2-year plan:
`2-year intro discount = yearly intro discount`

Formula for calculating multi-year discount:
`multi-year discount = full price - intro discount - coupon savings`

## Testing Instructions

* Use calypso live or checkout branch locally
* Go to e.g. http://calypso.localhost:3000/checkout/jetpack/jetpack_complete
* Notice that price discounts are broken down and displayed according to the design:

![CleanShot 2024-01-12 at 15 57 08@2x](https://github.com/Automattic/wp-calypso/assets/8419292/fbc338e9-d37c-4602-9b9a-35d7828b126a)


## Pre-merge Checklist
